code/modules/mining/machine_rigpress.dm

```diff
@@ -1,6 +1,9 @@
 /obj/machinery/mineral/rigpress
 	name = "hardsuit module press"
 	desc = "This machine converts certain items permanently into hardsuit modules."
+	desc_info = "The following devices can be made:\nJetpack = maneuvering jets\nOre detector = ore scanner module \
+	\nDrill = Hardsuit drill mount\nPlasma cutter = hardsuit plasma cutter\nDiamond pickaxe = hardsuit diamond drill mount \
```

Comment:
in theory it should be possible to automate this by adding a list on the rigpress that corresponds to the output, then updating desc_info during runtime